index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React</title>
   </head>
-  <body>
+  <body class="bg-gray-900 h-screen flex items-center justify-center">
```

Comment:
It's a template, move classes in the app. If you need to wrap your app, do this in `App.jsx`.